src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/RadioLabel/index.tsx

```diff
@@ -36,15 +39,20 @@ export const RadioLabelOverview = ({
   setStartDate,
   setEndDate,
 }: TreeChoiceWithFilteringProps): JSX.Element => {
+  const pathogen = useSelector(selectCurrentPathogen);
+  const {
+    overviewDescription = "",
+    overviewBestFor = "",
+    overviewGoodFor1 = "",
+    overviewGoodFor2 = "",
+  } = pathogen ? pathogenStrings[pathogen] : {};
```

Comment:
If pathogen is null, use an empty object to avoid errors related to trying to look up `null[someString]`